src/fortune-wheel-booth-frontend/src/utils/findPrize.ts

```diff
@@ -1,6 +1,11 @@
 import { PRIZES } from '../costants';
 
 export const findPrize = (option: string) => {
-  const prize = PRIZES.find((prize, index) => prize.option === option);
-  return prize ? PRIZES.indexOf(prize) : 99; // TODO: Fix this, 99 represents "no prize exists for this option"
+  if (option === 'merch') {
+    const merchNumbers = [1, 2, 3, 4, 5, 6];
```

Comment:
This is not so scalable, because if you change the prizes in the constants file you also have to change this array. We should rather keep this dynamic instead

Reply:
For example, getting all the ids of the merch options or the total count of the merch options inside the `PRIZES` array and extract randomly on those ids.
Not efficient but we're never going to have millions of prizes in the array I think